ansible/roles/postgresql_server/tasks/main.yml

```diff
@@ -46,6 +46,17 @@
     mode: '0755'
   when: postgresql_server__data_directory != '/var/lib/postgresql'
 
+
+- name: Make sure that custom PostgreSQL Log directory exists
+  file:
+    path: '{{ postgresql_server__log_directory }}'
+    state: 'directory'
+    owner: '{{ postgresql_server__user }}'
+    group: '{{ postgresql_server__group }}'
+    mode: '0755'
```

Comment:
Perhaps the directory more should be more strict (`0750`), I'm not sure if PostgreSQL emits confidential information in its logs...

Reply:
At least in my configuration it logs failed inserts and I use pretty much the defaults.

Reply:
Failure messages?
I might need to add the test for empty/undefined `postgresql_server__log_directory` as in my setup(s) I always have a log_directory defined

Reply:
> Failure messages?

Yes but this might be unrelated to this as I don't use this role to manage my postgres. Just wanted to give insights that this could happen.

Reply:
That is why I'd like to see what the error message(s) is that you are getting

Reply:
@hvisage I think that you changed the mode in the incorrect task. :-) Remember to update the mode in the other file as well.

Reply:
:blush: fixed the log directory permissions.

BUT: when you have the data directory with a 0755, then I'd make the case, you are already lax enough that a 0750 on the log directory isn't buying you any extra security, thus the need for the data to be as or stricter than the log directory.

Reply:
````
[40] STATEMENT:  INSERT INTO devices (user_id, device_id, display_name, hidden) VALUES('*******************', '**********', NULL, false)
````

I replaced the sensitive values with stars. It logged the full email address and device id.

Reply:
@SuperSandro2000 that is  unrelated to my PR which was ONLY adding the needed for a custom *LOG DIRECTORY* ;)
Perhaps a separate PR to handle that issue please ?

Reply:
Well, no as this container is totally unrelated to this project. I just wanted to answer the question is postgres can log sensitive data. Yes, it can.